examples/PhasorDynamics/Tiny/TwoBus/TwoBus_TGov1/CMakeLists.txt

```
@@ -1,12 +1,12 @@
-add_executable(phasordynamics_example1_tgov1 example1_tgov1.cpp)
-target_link_libraries(phasordynamics_example1_tgov1 
+add_executable(phasordynamics_example_twobus_tgov1 Example_TwoBus_TGov1.cpp)
+target_link_libraries(phasordynamics_example_twobus_tgov1 
```

Target names need to be detailed so we don't get target name clashes. But in this case we could do something like this:
```suggestion
add_executable(TwoBusTGov1.exe TwoBusTGov1.cpp)
target_link_libraries(TwoBusTGov1.exe
```
I understand we inherited untidy collection of names from before, but as we are reorganizing, it might be a good idea to set some rules that we will follow going forward.